src/GNURadioHandler.py

```diff
@@ -0,0 +1,44 @@
+from xmlrpc.client import ServerProxy
+
+
+class GNURadioHandler:
+
+	def __init__(self):	
+		self.server = ServerProxy('http://localhost:8080')
```

Comment:
Is this something that you think would be changed a lot (and would benefit from being exposed on the website side)? If so, is there any documentation for the 'http://localhost:8080' endpoint so that we can integrate this api endpoint directly on the website instead of relying on gs software for this?

Reply:
Hmm, good question. Would the website have to copy the methods in this file in that case? This is the doc for how it works in the gnuradio side: https://wiki.gnuradio.org/index.php/XMLRPC_Server

Reply:
The website should be capable of using this API yes. It's good that josh implemented this as a class, will make it easier to keep it as an object in the website, right?